ibm/service/vpc/data_source_ibm_is_backup_policy_plan.go

```diff
@@ -179,6 +221,19 @@ func dataSourceIBMIsBackupPolicyPlanRead(context context.Context, d *schema.Reso
 	if err = d.Set("name", backupPolicyPlan.Name); err != nil {
 		return diag.FromErr(fmt.Errorf("[ERROR] Error setting name: %s", err))
 	}
+	remoteCopyPolicies := []map[string]interface{}{}
+	if backupPolicyPlan.RemoteCopyPolicies != nil {
+		for _, remoteCopyPolicy := range backupPolicyPlan.RemoteCopyPolicies {
+			remoteCopyPoliciesMap, err := dataSourceIBMIsVPCBackupPolicyPlanRemoteCopyPolicyItemToMap(&remoteCopyPolicy)
+			if err != nil {
+				return diag.FromErr(err)
```

Comment:
Short explanation of the error would help here like in the above line 
"return diag.FromErr(fmt.Errorf("[ERROR] Error setting name: %s", err))"